bindings/index.ts

```diff
@@ -0,0 +1,15 @@
+import { setupCore } from './core';
+import { getSupportedMethods } from './helpers';
+import { setupCompile } from './compile';
+
+export default function setupBindings (solJson) {
+  const core = setupCore(solJson);
+  const compile = setupCompile(solJson, core);
+  const supported = getSupportedMethods(solJson);
```

Comment:
This might also be a good name:
```suggestion
  const methodFlags = getSupportedMethods(solJson);
```